hack/build/deploy_rhel.sh

```diff
@@ -16,29 +66,32 @@ go_build_args=(
 base_image="dynatrace-operator"
 out_image="${IMG:-quay.io/dynatrace/dynatrace-operator}:${TAG}"
 
+num_check='^[0-9]+$'
+if [[ -n "${QUAY_EXPIRY}" ]] && [[ "${QUAY_EXPIRY}" =~ ${num_check} ]]; then
+  expiry_args="--label quay.expires-after=${QUAY_EXPIRY}d"
+fi
+
 args="${go_build_args[@]}"
 if [[ "${LOCALBUILD}" ]]; then
   export CGO_ENABLED=1
   export GOOS=linux
   export GOARCH=${GOARCH:-amd64}
 
-  go build -ldflags "$args" -tags exclude_graphdriver_btrfs -o ./build/_output/bin/dynatrace-operator ./src/cmd/operator/
-
-  if [[ "$?" != 0 ]]; then
-	  echo "ERROR: go build exited abnormally. Aborting..."
-	  exit 10
-  fi
-
+  go build -ldflags "${args}" -tags exclude_graphdriver_btrfs -o ./build/_output/bin/dynatrace-operator ./src/cmd/
+  check_err GO_BUILD
+  
   go get github.com/google/go-licenses
   go-licenses save ./... --save_path third_party_licenses --force
+  check_err GO_LICENSES
 
-  docker build . -f ./Dockerfile-localbuild -t "${base_image}" --label "quay.expires-after=14d" --no-cache
+  docker build . -f ./Dockerfile-localbuild -t "${base_image}" ${expiry_args} --no-cache
```

Comment:
do you use the local build? the `Dockerfile-localbuild` is pretty outdated and I was planning on removing it.

Reply:
TL;DR: do with Dockerfile-localbuild as you like.

Porting to power/z architectures is on hiatus, so I've sort of stopped working on it. I was using the localbuild Dockerfile, with intent to either fork or modify the existing Dockerfile to support s390x/ppc64le (with the build stage on UBI, since dynamic linking is required). 

The development VMs I was using were configured more as pets than cattle, per se, and the golang packages were too old to use ubi8/go-toolset for a UBI-based build.